lib/process-services-cloud/src/lib/process-services-cloud.module.ts

```diff
@@ -46,7 +51,10 @@ import { UserPreferenceCloudService } from './services/user-preference.cloud.ser
             }
         },
         BaseCloudService,
-        UserPreferenceCloudService
+        UserPreferenceCloudService,
+        LocalPreferenceCloudService,
+        { provide: PROCESS_FILTERS_SERVICE_TOKEN, useClass: LocalPreferenceCloudService },
```

Comment:
Looks complicated and may require additional docs and explanations to the developers

Reply:
Well by default the module will inject a LocalPreferenceCloudService. 
On the documentation, we can explain how to override that to use the UserPreferenceCloudService

Reply:
Why not using it like the following?
```
{ provide: UserPreferenceCloudService, useClass: LocalPreferenceCloudService }
```

Reply:
I think the reason is the same as above. Maybe we should agree on a rule here, either we want to use tokens or not, but whatever the decision is, adhere to that.